vuepress/docs/next/tutorials/consume/mt-cds.md

```diff
@@ -0,0 +1,285 @@
+
+    nginx.org/client-max-body-size: "150m" # edit according to the file size you require
+spec:
+  $INGRESS_CLASS_FIELD
+  rules:
+    - host: cds-YOUR-APP-NAME.YOUR-HOST-ADDRESS
+      http:
+        paths:
+          - backend:
+              service:
+                name: YOUR-APP-NAME-cds-YOUR-TENANT-CODE-service
+                port:
+                  number: 8081
+            pathType: Prefix
+            path: /$TN
+          - backend:
+              service:
+                name: $APP-cds-$TN-service
+                port:
+                  number: 8080
+            pathType: Prefix
+            path: /api/v1/
+#  tls: # Optional; if used, the user needs to customize the Entando operator ConfigMap to also use TLS
+#    - hosts:
+#        - cds-$APP.$HOST
+#      secretName: cds-tls
+```
+
+<details><summary><b>Alternate ConfigMap for TLS</b></summary>
+
+This example ConfigMap is suggested for direct integration on Kubernetes.
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: entando-operator-config
+data:
+  entando.pod.completion.timeout.seconds: "2000"
+  entando.pod.readiness.timeout.seconds: "2000"
+  entando.requires.filesystem.group.override: "true"
+  entando.ingress.class: "nginx"
+  entando.tls.secret.name: YOUR-TLS-SECRET
+
+# More..
+#  entando.k8s.operator.image.pull.secrets: sample-pull-secret
+#  entando.docker.registry.override: "docker.io"
+#  entando.ca.secret.name: sample-ca-cert-secret
+#  entando.assume.external.https.provider: "true"
+#  entando.k8s.operator.impose.limits: "true"
+```
+
+</details>
+
+## Apply the Descriptors
+Install the CDS descriptors in the order listed below.
+``` bash
+kubectl apply -f YOUR-CDS-PVC.yaml
+kubectl apply -f YOUR-CDS-KEYCLOAK-SECRET.yaml
+kubectl apply -f YOUR-CDS-DEPLOYMENT.yaml
+kubectl apply -f YOUR-CDS-INGRESS.yaml
+```
+**can these be done at once**
+
+## Create the Assets Archive 
+Create an archive of the resources and assets for your application.  Within it, the resources folder must be named `public` and sensitive information should be stored in the `private` folder.
+
+## APIs
+Create API requests to include these fields:
+ * `CDS_PATH` : # the base path of the new CDS
+ * `KC_URI` : # the url of token service of the tenant1 realm
+ * `KC_CLIENT_ID` : # the clientId of the reserved client used by the entando-de-app in tenant1 realm
+ **what is the reserved client???**
+ * `KC_CLIENT_SECRET` : # the secret of that client
+
+## Configure the Entando App Engine
+1. Scale to 0 the de-app deployment
+2. Open `entando-de-app` and add these environment variables using the :
+``` yaml
+spec:
+   env:
+     - name: CDS_ENABLED
+       value: "true"
+     - name: CDS_PUBLIC_URL
+       value: http://cds.YOUR-HOSTNAME/primary
+     - name: CDS_PRIVATE_URL
+       value: http://cds:8080
+     - name: CDS_PATH
+       value: /api/v1
+```
+3. Delete the following two volume specifications related to the quickstart deployment:
```

Comment:
Is this still have to be done even if we are not using the quickstart installation method? I think so but the volume names would not follow that pattern `quickstart...` we should use placeholders too.

Reply:
edited the quickstart names with placeholders